templates/exercise.hbs

```diff
@@ -0,0 +1,8 @@
+// {{exercise}}.js
+// Add description about explanation for exercise
+
+// Execute `yarn jslings hint {{exercise}}` for hints!
+
```

Comment:
```suggestion
// DO NOT EDIT ABOVE THIS LINE
```

Adding this comment will ensure that people don't accidentally delete the description as this is the only place we are storing the descriptions on the exercise

Reply:
I'm wondering if this is necessary? It adds extra distraction in our files (`// DO NOT EDIT...`). If a PR is open that doesn't contain a proper description or the hint line, we simply wouldn't merge the PR?

Reply:
I meant it more so for the user. 

Isn't this the file where the user would be writing the code? We don't want the user accidentally deleting the description simply because we have no other way as of now to regenerate the description.

Reply:
Added it! 🚀